sunpy/net/jsoc/jsoc.py

```diff
@@ -897,3 +897,10 @@ def _can_handle_query(cls, *query):
     @classmethod
     def _attrs_module(cls):
         return 'jsoc', 'sunpy.net.jsoc.attrs'
+
+    @classmethod
+    def register_values(cls):
+        from sunpy.net import attrs
+        # TODO Implement this
```

Comment:
Could you open an issue for this TODO?

Reply:
https://github.com/sunpy/sunpy/issues/4083